### DISCUSSION TITLE
Is it possible to break lines in equations like `\allowdisplaybreaks`

### DISCUSSION THREAD

**Discussion by Zeta611** [typst/typst#4291] [Q&A] [answered]
With LaTeX + `amsmath`, it is possible to span `align`ed equations into multiple pages via `\allowdisplaybreaks`.
Is this possible in typst?

Thanks!

**Comment by Enivex** [comment, marked as answer]
Not out of the box yet, but as far as I'm aware the equate package has a workaround.

**Reply by Zeta611** [reply]
Wow. Thank you for such a quick reply! And it works like charm.
